From: Jiri Slaby <jslaby@suse.cz>
Date: Tue, 18 Oct 2016 17:48:37 +0200
Subject: Revert "tty: serial: 8250: add CON_CONSDEV to flags"
Patch-mainline: not yet, perhaps a fix will be queued, to be discussed yet
References: bnc#1003623

This reverts commit d03516df837587368fc6e75591f6329c072b9eb5 as it
breaks console= handling.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 drivers/tty/serial/8250/8250_core.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/tty/serial/8250/8250_core.c b/drivers/tty/serial/8250/8250_core.c
index 240a361b674f..e8819aa20415 100644
--- a/drivers/tty/serial/8250/8250_core.c
+++ b/drivers/tty/serial/8250/8250_core.c
@@ -675,7 +675,7 @@ static struct console univ8250_console = {
 	.device		= uart_console_device,
 	.setup		= univ8250_console_setup,
 	.match		= univ8250_console_match,
-	.flags		= CON_PRINTBUFFER | CON_ANYTIME | CON_CONSDEV,
+	.flags		= CON_PRINTBUFFER | CON_ANYTIME,
 	.index		= -1,
 	.data		= &serial8250_reg,
 };
-- 
2.10.1

